### DISCUSSION TITLE
RomM working on Unraid

### DISCUSSION THREAD

**Discussion by sfumat0** [rommapp/romm#289] [Q&A]
I have seen a user reporting RomM working well on Unraid. I am attempting to set this up but failing. Is there a simple guide?
I have used the docker-compose.example.yml to add in the variables, ports and paths but it doesn't seem to be creating the db or creating the config.

This is my first post so please forgive me if this is not the right place to request support.

**Comment by gantoine** [comment]
Moved to issue #290
